spec/features/collection_drafts/forms/help_text_spec.rb

```diff
@@ -111,5 +111,35 @@
         expect(page).to have_link('View More Information', href: 'https://wiki.earthdata.nasa.gov/display/CMR/Version')
       end
     end
+
+    context 'when clicking on the help icon with a help link' do
+      before do
+        within '.metadata' do
+          click_on 'Acquisition Information'
+        end
+
+        click_on 'Help modal for Platforms'
+      end
+
+      it 'displays the help link in the modal' do
+        expect(page).to have_link('View More Information', href: 'https://wiki.earthdata.nasa.gov/display/CMR/Platform')
+      end
+    end
+
+    # context 'when clicking on the help icon with a help link' do
```

Comment:
I'm not sure why this test is commented out -- I thought we discussed removing the test above and adding this one

Reply:
Still need to work on the test bellow for Data Contacts, the selector doesn't work yet, but checked in because switch to another ticket to work on.